src/sql/workbench/contrib/notebook/test/electron-browser/cell.test.ts

```diff
@@ -833,4 +833,39 @@ suite('Cell Model', function (): void {
 			assert.strictEqual(actualMsg, testMsg);
 		});
 	});
+
+	test('Should emit event on markdown cell edit', async function (): Promise<void> {
+		let notebookModel = new NotebookModelStub({
+			name: '',
+			version: '',
+			mimetype: ''
+		});
+		let contents: nb.ICellContents = {
+			cell_type: CellTypes.Markdown,
+			source: ''
+		};
+		let model = factory.createCell(contents, { notebook: notebookModel, isTrusted: false });
+		assert(!model.isEditMode);
+
+		let createCellModePromise = () => {
+			return new Promise((resolve, reject) => {
+				setTimeout(() => reject(), 2000);
```

Comment:
If this promise is rejected, can we emit a message where we can quickly debug this issue?